steps/appointee/same-address/SameAddress.js

```diff
@@ -23,10 +24,12 @@ class SameAddress extends SaveToDraftStore {
   }
 
   answers() {
+    const content = require(`./content.${i18next.language}`);
```

Comment:
Can you please move this line outside of the method

Reply:
That wouldn't work as `i18next` is still not initialised at that point.